diff -up DateTimeBundle/DateTime-0.53/leaptab.txt.leap DateTimeBundle/DateTime-0.53/leaptab.txt
--- DateTimeBundle/DateTime-0.53/leaptab.txt.leap	2016-08-24 13:51:03.855023182 +0200
+++ DateTimeBundle/DateTime-0.53/leaptab.txt	2016-08-24 13:56:47.441527654 +0200
@@ -24,3 +24,4 @@
 2009  Jan. 1  +1
 2012  Jul. 1  +1
 2015  Jul. 1  +1
+2017  Jan. 1  +1
diff -up DateTimeBundle/DateTime-0.53/lib/DateTime/LeapSecond.pm.leap DateTimeBundle/DateTime-0.53/lib/DateTime/LeapSecond.pm
--- DateTimeBundle/DateTime-0.53/lib/DateTime/LeapSecond.pm.leap	2016-08-24 13:51:23.222940269 +0200
+++ DateTimeBundle/DateTime-0.53/lib/DateTime/LeapSecond.pm	2016-08-24 13:57:05.702434944 +0200
@@ -109,6 +109,7 @@ sub _initialize {
 2009  Jan. 1  +1
 2012  Jun. 1  +1
 2015  Jul. 1  +1
+2017  Jan. 1  +1
     ) );
 }
 
diff -up DateTimeBundle/DateTime-0.53/t/19leap_second.t.leap DateTimeBundle/DateTime-0.53/t/19leap_second.t
--- DateTimeBundle/DateTime-0.53/t/19leap_second.t.leap	2016-08-24 13:58:49.156909707 +0200
+++ DateTimeBundle/DateTime-0.53/t/19leap_second.t	2016-08-24 13:58:15.279081704 +0200
@@ -2,7 +2,7 @@
 
 use strict;
 
-use Test::More tests => 174;
+use Test::More tests => 175;
 use DateTime;
 
 # tests using UTC times
@@ -950,6 +950,7 @@ use DateTime;
                    [ 2008, 12, 31 ],
                    [ 2012, 6,  30 ],
                    [ 2015, 6,  30 ],
+                   [ 2016, 12, 31 ],
                  )
     {
         my $dt = eval { DateTime->new( year   => $date->[0],
